dali/test/python/test_backend_impl.py

```diff
@@ -142,3 +142,18 @@ def check_squeeze(shape, dim, in_layout, expected_out_layout):
              (0, (1, 5, 1), "ABC", "BC"),
              (None, (3, 5, 1), "ABC", "AB")]:
         yield check_squeeze, shape, dim, in_layout, expected_out_layout
+
+
+def test_tensorlist_constructor_from_list_of_tensors():
```

Comment:
It might be nice to test non-uniformly shaped TensorList as well (where each tensor has different dimensions).

Reply:
done